seccomp/README.md

```diff
@@ -128,49 +292,48 @@ Profiles can contain more granular filters based on the value of the arguments t
 * `value` is a parameter for the operation
 * `valueTwo` is used only for SCMP_CMP_MASKED_EQ
 
-The rule matches if **all** args match. To achieve the effect of an or, add multiple rules.
+The rule only matches if **all** args match. Add multiple rules to achieve the effect of an OR.   << **NOTE TO LAB OWNER: IS THAT STATEMENT CORRECT?**
 
-Strace can be used to get a list of all system calls made by a program.
+`Strace` can be used to get a list of all system calls made by a program.
 It's a very good starting point for writing seccomp policies.
 Here's an example of how we can list all system calls made by `ls`:
 
 ```
 $ strace -c -f -S name ls 2>&1 1>/dev/null | tail -n +3 | head -n -2 | awk '{print $(NF)}'
+access
+arch_prctl
+brk
+close
+execve
+<SNIP>
+statfs
+write
 ```
 
-### Demo
+The output above shows all of the syscalls that will need enabling for a container running the `ls` program to work.
 
-By copying the default docker profile and then removing system calls selectively we can show that we are able to intercept system calls with seccomp. For this demo we've removed chmod, fchmod and chmodat from the default profile.
+In this step you learned the format and syntax of Docker seccomp profiles. You also learned the order of preference for actions, as well as how to determine the syscalls needed by an individual program.
 
-```
-$ docker run --rm -it --security-opt seccomp=default-no-chmod.json alpine chmod 777 /
-```
+# <a name="test"></a>Step 6: A few gotchas
 
-### Gotchas
+The remainder of this lab will walk you through a few things that are easy to miss when using seccomp with Docker.
 
-There are some things that you can easily miss when using seccomp with docker.
+#### Timing of a seccomp profile application
 
-#### Timing of application of seccomp policies
+In versions of Docker prior to 1.12, seccomp polices tended to be applied very early in the container creation process. This resulted in you needing to add syscalls to your profile that were required for the container creation process but not required by your container. This was not ideal. See:
 
-In versions before 1.12 seccomp polices tend to be applied too soon.
-This means that you may need to add system calls that your application doesn't use in order for the container to be started successfully.
-This behavior hasn't been thoroughly documented yet. See: https://github.com/docker/docker/issues/22252 https://github.com/opencontainers/runc/pull/789
+- https://github.com/docker/docker/issues/22252
+- https://github.com/opencontainers/runc/pull/789
 
-The best way to avoid this issue is to use
-
-```
---security-opt no-new-privileges
-```
-
-Note that this also disables gaining privileges through setuid binaries.
+A good way to avoid this issue can be to use the `--security-opt no-new-privileges` flag when starting your container. However, this will also prevent you from gaining privileges through `setuid` binaries.
```

Comment:
We should also mention that this helps only in docker 1.12 ( @justincormack please confirm )